app/models/invitation.rb

```diff
@@ -1,7 +1,10 @@
  class Invitation < ActiveRecord::Base
-  belongs_to :review
   attr_accessible :email, :username
 
+  belongs_to :review
+  has_one    :reviewee, through: :review, source: :associate_consultant
+  has_many   :feedbacks, through: :review, source: :feedbacks

```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.